admob-latest/src/main/java/com/rakuten/tech/mobile/miniapp/ads/MiniAppH5AdsWebViewClient.kt

```diff
@@ -0,0 +1,14 @@
+package com.rakuten.tech.mobile.miniapp.ads
+
+import android.content.Context
+import android.webkit.WebView
+import android.webkit.WebViewClient
+import com.google.android.gms.ads.h5.H5AdsWebViewClient
+
+class MiniAppH5AdsWebViewClient(private val context: Context) {
```

Comment:
I think, we can rename this function as following.

```suggestion
/**
 * The class for providing functions related to H5AdsWebViewClient for using in a WebView.
 */
class MiniAppH5AdsProvider {
```